packages/patternfly-4/react-core/src/components/Card/CardHeader.tsx

```diff
@@ -5,18 +5,17 @@ import titleStyles from '@patternfly/react-styles/css/components/Title/title';
 
 export interface CardHeaderProps extends React.HTMLProps<HTMLDivElement> {
     /** Content rendered inside the Card Footer */
-    children?: React.ReactNode; 
+    children?: React.ReactNode;
```

Comment:
It's interesting that TSLint is ok with 4-space indentation here and 2-space indentation everywhere else. Maybe that's just something it can't automatically --fix?